deepchem/trans/transformers.py

```diff
@@ -1318,6 +1329,36 @@ def gaussian_blur(self, sigma=0.2):
     """
     return scipy.ndimage.gaussian_filter(self.Image, sigma)
 
+  def crop(self, x_crop, y_crop):
+    """ Crops the image from the center
+
+    Parameters
+    ----------
+    x_crop - bound for x.
+    y_crop - bound for y.
```

Comment:
A few thoughts on this:

1. Could you make these a little more descriptive?  It isn't obvious what "bound for x" means.
2. There are several ways you might want to specify what to crop.  The one you've chosen (total number of pixels to remove on each axis) is one useful one, but not the only one.  Other useful options would be the size of the cropped image, and the number of pixels to remove from each side.  The latter could be specified as one number (remove the same number of pixels from every side), or four numbers (remove different numbers of pixels from every side).  Perhaps we should have multiple named arguments you can specify, or even multiple methods, to allow the user to pick whichever method is most convenient for them?
3. It still isn't using Numpy format for the comments.

For example, we might have one method called `crop()` with these arguments:

```
top: int
    the number of pixels to remove from the top of the image
left: int
    the number of pixels to remove from the left of the image
bottom: int
    the number of pixels to remove from the bottom of the image
right: int
    the number of pixels to remove from the right of the image
```

and another called `center_crop()` with these arguments:

```
width: int
    the width of the cropped image in pixels
height: int
    the height of the cropped image in pixels
x_crop: int
    the total number of pixels to remove in the horizontal direction,
    evenly split between the left and right sides
y_crop: int
    the total number of pixels to remove in the vertical direction,
    evenly split between the top and bottom sides
```

All four of those would have default values of `None`.  The user would be expected to specify either `width` or `x_crop`, but not both, and same for the height.